NuGet.Config

```diff
@@ -45,27 +46,28 @@
       <package pattern="xunit.*" />
       <package pattern="xunitxml.testlogger" />
     </packageSource>
-    <packageSource key = "dotnet-eng">
+    <packageSource key="dotnet-eng">
       <package pattern="Microsoft.DotNet.*" />
     </packageSource>
-    <packageSource key = "nuget-build">
+    <packageSource key="nuget-build">
       <package pattern="nuget.client.endtoend.testdata" />
       <package pattern="nugetvalidator" />
     </packageSource>
-    <packageSource key = "vside">
+    <packageSource key="vside">
       <package pattern="envdte" />
       <package pattern="envdte100" />
       <package pattern="envdte80" />
       <package pattern="envdte90" />
       <package pattern="envdte90a" />
+      <package pattern="Microsoft.*" />
```

Comment:
no, please don't add an identical pattern that exists in another package source. We had problems in the past when trying to upgrade the VS SDK, because the vside feed had packages not on nuget.org, and I don't remember the details, but it was problematic.

It's precisely why there are so many other patterns below starting with `Microsoft.`